src/query/vector_distance_enum.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * vector_distance_enum.h - vector distance metric enum
+ */
+
+#ifndef _VECTOR_DISTANCE_ENUM_H_
+#define _VECTOR_DISTANCE_ENUM_H_
+
+enum DB_VECTOR_DISTANCE_METRIC
+{
+  COSINE = 0,
+  EUCLIDEAN = 1,
```

Comment:
스펙에서 지원 가능한 metric을 모두 넣어두고, 주석으로 막아 두시죠.
```suggestion
  //COSINE = 0,
  //DOT = 1,
  EUCLIDEAN = 2,
  //EUCLIDEAN_SQUARED = 3,
  //HAMMING = 4,
  //MANHATTAN = 5,
  //JACCARD = 6
```